README.md

```diff
@@ -9,6 +9,9 @@ graph and individual task configurations, will perform task scheduling and memor
 user interface so that the end user can relinquish scheduling to the runtime (default behavior) or take full control of scheduling and
 mapping, if desired. The target audience for ATMI is application programmers or middleware developers for high-level languages.
 
+## Deprecation Notice ##
+ATMI is not being used for active development after ROCm 5.6. Previously released branches are still available for reference.
```

Comment:
Can we word it slightly differently like below? We want to maintain it at the AMDResearch GitHub location (https://github.com/AMDResearch/atmi) as a Research project. 

> ATMI is not being released as part of AMD ROCm software after ROCm 5.6. Previously released branches are still available for reference. ATMI will henceforth be maintained as a separate research project at https://github.com/AMDResearch/atmi.